digdag-docs/src/operators/py.md

```diff
@@ -76,6 +76,40 @@ See [Python API documents](../python_api.html) for details including variable ma
     print("simple execution")
   ```
 
+  You can pass arguments to class for initialization using `_export` as the following:
```

Comment:
Only describe the way to use `_export` is not enough.
Could you add describe the way to pass argument as task parameters.
```
+a:
  py>: tasks.python_args.required_arguments
  required1: "1"
  required2: 2
```

Reply:
Sorry for the late confirmation. Thank you for your review and detailed instructions. Please check my modification.
- modify example to describe what data type the passed arguments will actually have inside Python
- add description the way to pass arguments as task parameters
- add example about how to pass arguments for class and method simultaneously